pkgs/top-level/python-packages.nix

```diff
@@ -8247,6 +8247,24 @@ in {
     inherit (pkgs) which;
   };
 
+  nixpkgs = buildPythonPackage rec {
+    disabled = ! pythonAtLeast "3.5";
+    pname = "nixpkgs";
+    version = "0.2.1";
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0xrdk3n0jh8gwqflw9f4pj5dj43kq7ihdarmy396f95a353grp96";
+    };
+    buildInputs = with pkgs.python3Packages; [ pbr ];
```

Comment:
Don't reference python inputs by using `pkgs.python3Packages` since `python3Packages` may be pointing at a package set for a different interpreter.
This would break the following code: `python35.withPackages(ps: [ ps.nixpkgs ])` since this would mix dependencies from python3.5 and python3.6.

Use `buildInputs = with self [ pbr ]` instead

Reply:
OK.